src/rpc/register.h

```diff
@@ -21,6 +21,8 @@ void RegisterMiningRPCCommands(CRPCTable &tableRPC);
 void RegisterRawTransactionRPCCommands(CRPCTable &tableRPC);
 /** Register raw transaction RPC commands */
 void RegisterSignerRPCCommands(CRPCTable &tableRPC);
+/** ELEMENTS: Register exchange rates RPC commands */
+void RegisterExchangeRatesRPCCommands(CRPCTable &tableRPC);
 
 static inline void RegisterAllCoreRPCCommands(CRPCTable &t)
 {
```

Comment:
Don't you need to (conditionally) call RegisterExchangeRatesRPCCommands in RegisterAllCoreRPCCommands ?

Reply:
Do we have a test case for these rpc commands yet?

Reply:
> Don't you need to (conditionally) call RegisterExchangeRatesRPCCommands in RegisterAllCoreRPCCommands ?

https://github.com/SequentiaSEQ/SEQ-Core-Elements/pull/10/files#diff-b1e19192258d83199d8adaa5ac31f067af98f63554bfdd679bd8e8073815e69dR1337 already does this. And I wouldn't consider an Elements-only optional feature part of the "Core RPCs".

Reply:
> Do we have a test case for these rpc commands yet?

For `setfeeexchangerates`, yes, but not `getfeeexchangerates`. I'll add a sanity test for it.